fuzz/CMakeLists.txt

```diff
@@ -17,58 +17,58 @@ add_link_options(-fsanitize=address,fuzzer,undefined)
 
 #target_include_directories(h265_profile_tier_level_parser_fuzzer PUBLIC /usr/local/include ${CMAKE_CURRENT_SOURCE_DIR} webrtc)
 
-add_executable(h265_utils_fuzzer h265_utils_fuzzer.cc)
+add_library(h265_utils_fuzzer h265_utils_fuzzer.cc)
```

Comment:
I'll try your patch later, but I need to keep building the fuzzer binaries.

Reply:
This is turning the fuzzer binaries (executables) into libraries. 

I need the fuzzer binaries to fuzz the tool

Reply:
MSVC attempts to link executables with no main function and fails, any suggestions?
```
unresolved external symbol main referenced in function "int __cdecl invoke_main(void)"
```

Reply:
I'm ok if there is no fuzzing in windows. Maybe you can add an ifdef around all the fuzzer executable definitions in the CMakeLists.txt ?

Reply:
Would that mean building the libs on windows is not needed at all? Or is it still useful to build the libs there and executables on linux platforms?

Reply:
I pushed changes, everything still works, no fuzzer libs or exe on windows. Maybe I will look more into getting that working in the future if you think it is valuable.